canopen/sdo/server.py

```diff
@@ -123,7 +123,6 @@ def block_download(self, data):
         self.abort(0x05040001)
 
     def init_download(self, request):
-        # TODO: Check if writable
```

Comment:
Now I remember what this was for. Preferably we should check if it is writable before accepting segmented data. Right now a segmented download won't fail until the data has been fully received. So you can just leave the TODO for now.

Reply:
re-added the comment